src/treemap.ts

```diff
@@ -102,6 +105,9 @@ function px(x: number): string {
 function defaultOptions(options: Partial<Options>): Options {
   const opts = {
     padding: options.padding || [14, 3, 3, 3],
+    lowerBound: options.lowerBound === undefined ? 0.1 : options.lowerBound,
```

Comment:
Is this the same:
options.lowerBound ?? 0.1

Reply:
Yes it is. If you want I can attempt to upgrade the ts version to 3.7+ but when I attempted it originally there were other errors being thrown by the ts compiler. I think that a separate PR for a ts version bump is more appropriate.

Reply:
Oh yeah, sorry!  My mind is in my work where we're already using 3.7 :)